packages/flutter_tools/test/commands.shard/hermetic/build_ios_framework_test.dart

```diff
@@ -62,6 +56,7 @@ void main() {
           fileSystem: memoryFileSystem,
           processManager: FakeProcessManager.any(),
         );
+
```

Comment:
Nit: revert

Reply:
> Can you rebase this to resolve the current test failures? Thanks!

Sorry, you lost me 🙈.  Do you want me to rebase, or revert and what commit are we talking about? 
Can you be more specific please? 😊

Reply:
Oh sorry to be confusing. I meant to ask to revert any unnecessary formatting changes, and to rebase your PR with the latest from master.
The failing `customer_testing` shard tends to have issues as a PR gets older. Merging in updates from master don't work to resolve it when this happens, it requires rebasing. I hope this is a bit clearer. Let me know if it is not. :)

Reply:
Thank you! Now I get it =)